docs/lib/auth/fragments/js/customui.md

```diff
@@ -1,25 +1,6 @@
 ### Customize UI theme
 
-For React, you can create your own theme and use it to render Amplify components:
-
-Your custom theme must use the selectors from the following [template](https://github.com/aws-amplify/amplify-js/blob/main/packages/aws-amplify-react/src/Amplify-UI/Amplify-UI-Theme.tsx).
-
-```javascript
-import MyTheme from './MyTheme';
-
-<Authenticator theme={MyTheme} />
-```
-
-Alternatively, you can change a few properties and pass in a theme object from the same file:
-
-```javascript
-const MyTheme = {
-    signInButtonIcon: { 'display': 'none' },
-    googleSignInButton: { 'backgroundColor': 'red', 'borderColor': 'red' }
-}
-
-<Authenticator theme={MyTheme} />
-```
+For React, you can use CSS Variables as described [here](~/ui/customization/theming.md/q/framework/react).
```

Comment:
This should this include other frameworks, shouldn't it?  I just didn't know where to link it to...
```suggestion
For web frameworks, you can use CSS Variables as described [here](~/ui/customization/theming.md/q/framework/react).
```

Reply:
Theoretically it could link to just `~/ui/customization/theming.md`, which has the grid of framework options.

Reply:
Yeah, good call! I think `~/ui/customization/theming.md` is good to redirect user to their framework (if they selected one during their visit).